### PR TITLE
Update mitigation-of-hot-issues.md

### PR DESCRIPTION
**Pull request by jinwhu**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases

### CODE REVIEW

File: zh/tidb-database-development-specification/mitigation-of-hot-issues.md [hunk 1]
```diff
@@ -30,13 +30,16 @@ TiDB 发生写入热点的原因主要有以下几种：
 
 1. 配置 `SHARD_ROW_ID_BITS` 参数打散写入热点
 
-    对于非索引组织表（默认行为），TiDB 会使用一个隐式的自增 rowid，大量 INSERT 时会把数据集中写入单个 Region，造成写入热点。 通过设置 `SHARD_ROW_ID_BITS` 可以把 rowid 打散写入多个不同的 Region，缓解写入热点问题。 但是设置的过大会造成 RPC 请求数放大，增加 CPU 和网络开销。
+    对于非索引组织表（默认行为），TiDB 会使用一个隐式的自增 rowid，大量 INSERT 时会把数据集中写入单个 Region，造成写入热点。 通过设置 `SHARD_ROW_ID_BITS` 可以把 rowid 打散写入多个不同的 Region，缓解写入热点问题。 
+    但是设置的过大会造成 RPC 请求数放大，增加 CPU 和网络开销。需注意：通过其他方式设计的随机主键，当主键过分打散时，也会存在同样的问题。
+    其他方式生成的过于随机的主键（比如使用 UUID）也会有此问题。这是因为尽管其离散程度很高，但是也会因此带来网络层低效攒批、RocksDB compaction时的写入放大、Regions的过量激活问题，给整体NETWORK 和 IO 和 CPU 都带来压力。
```
Comment (jackysp) [inline zh/tidb-database-development-specification/mitigation-of-hot-issues.md:35]:
```suggestion
    其他方式生成的过于随机的主键（比如使用 UUID）也会有此问题。这是因为尽管其离散程度很高，但是也会因此带来网络层低效攒批、RocksDB compaction 时的写入放大、Regions的 过量激活问题，给整体 NETWORK 和 IO 和 CPU 都带来压力。    
```

**Review by jackysp — APPROVED** [review]
LGTM

**Comment by jackysp** [comment]
PTAL @TomShawn

**Comment by TomShawn** [comment]
@jackysp Please try to merge it again. Thanks~

**Pull request merged**